content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -25,15 +59,20 @@ Search for QuestionnaireResponses that meet supplied query parameters:
 
     GET /QuestionnaireResponse?:parameters
 
+_Implementation Notes_
+
+* QuestionnaireResponses are sometimes system generated without any answers. These may be updated with the correct responses at any time.
+
 ### Authorization Types
 
 <%= authorization_types(provider: true, patient: true, system: true) %>
```

Comment:
make patient: false for now

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/495a781b4b4b3edf9bfa48de95a6395c6ff2c9fe